### PR TITLE
fix bug of delete service route

### PR DESCRIPTION
**Pull request by ymybxx**
The branch of 0.2.0 is master-0.2.0. If your PR is to solve the problem of 0.2.0, please submit the PR to it.
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

删除服务级别的条件路由，实际并没有删除掉zk中provider/routers/下的内容 
原因：添加的dubbo url路径带version，而删除的dubbo url没有version

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist.
* [ ] Run `mvn clean compile --batch-mode -DskipTests=false -Dcheckstyle.skip=false -Drat.skip=false -Dmaven.javadoc.skip=true to make sure basic checks pass.

### CODE REVIEW

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/RouteUtils.java [hunk 1]
```diff
@@ -230,7 +230,7 @@ public static RoutingRule insertConditionRule(RoutingRule existRule, ConditionRo
                 existRule.setKey(conditionRoute.getApplication());
                 existRule.setScope(Constants.APPLICATION);
             } else {
-                existRule.setKey(conditionRoute.getService().replace("/", "*"));
+                existRule.setKey(buildServiceKey(conditionRoute));
```
Comment (haoyann) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/RouteUtils.java:233]: 感谢你的贡献，但是我改变这里的内容，会影响到更新方法。我的想法是在删除和更新的时候手动设置一下 route 的 servcie 字段为id。
示例：
`org.apache.dubbo.admin.service.impl.RouteServiceImpl#deleteConditionRoute`
```
    old.setService(id);
    URL oldUrl = old.toUrl();
```

Comment (ymybxx) [reply dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/RouteUtils.java]: 的确，而且我的方法无法解决已在使用用户得问题。代码已经修改提交了

Comment (haoyann) [reply dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/RouteUtils.java]: 请把黑白名单的也修改一下。
https://github.com/apache/dubbo-admin/blob/bc280e52cc471f58bf3bfd76fcefc5ce8bbb7772/dubbo-admin-server/src/main/java/org/apache/dubbo/admin/service/impl/RouteServiceImpl.java#L211
https://github.com/apache/dubbo-admin/blob/bc280e52cc471f58bf3bfd76fcefc5ce8bbb7772/dubbo-admin-server/src/main/java/org/apache/dubbo/admin/service/impl/RouteServiceImpl.java#L130

Comment (ymybxx) [reply dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/RouteUtils.java]: 黑白名单原本就是按照id更新路由的，测过没有问题

**Pull request merged**